tests/framework/db/mysql/CommandTest.php

```
@@ -8,19 +8,136 @@
+        self::assertEmpty(
+            $schema->getTableSchema($tableName, true)->getColumn($columnName)->comment,
+            'Column comment must be removed.',
+        );
+
+        DbHelper::dropTablesIfExist($db, [$tableName]);
+    }
+
+    #[DataProviderExternal(CommandProvider::class, 'commentSpecialCharacters')]
+    public function testAddCommentOnColumnRoundTripsSpecialCharacters(string $comment): void
+    {
+        $db = $this->getConnection(false);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)